extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -86,11 +88,12 @@ function ($, w, mw, loader, nirvana, tracker, throbber, labeling) {
 		}
 
 		// Mark selected type
-		$preselectedType = $classificationForm.find('#template-classification-' + templateType);
+		preselectedType = templateType;
+		$preselectedTypeInput = $classificationForm.find('#template-classification-' + preselectedType);
 
-		if ($preselectedType.length !== 0) {
+		if ($preselectedTypeInput.length !== 0) {
 			$classificationForm.find('input[checked="checked"]').removeAttr('checked');
-			$preselectedType.attr('checked', 'checked');
+			$preselectedTypeInput.attr('checked', 'checked');
```

Comment:
Yeah, I tried it but strangely - it does not check the selected radio button. Do you think I should investigate further or leave for a future improvement (which may not happen) :P